t/e2e/whisper/whisper_jail_test.go

```diff
@@ -192,21 +187,23 @@ func (s *WhisperJailTestSuite) TestJailWhisper() {
 					topics: [topic],
 					symKeyID: keyid
 				});
-
+				// creating a filter is an async operation
+				setTimeout(function() {
 				// post message
-				var message = {
+				  var message = {
 					ttl: 10,
 					powTarget: 0.001,
 					powTime: 2,
 					topic: topic,
 					symKeyID: keyid,
-			  		payload: web3.toHex(payload),
-				};
+					payload: web3.toHex(payload),
+				  };
 
-				var sent = shh.post(message)
-				if (!sent) {
+				  var sent = shh.post(message)
+				  if (!sent) {
 					throw 'message not sent: ' + JSON.stringify(message);
-				}
+				  }
+				}, 100)
```

Comment:
@dshulyak Out of curiosity, have you tried using a value of `0` instead of `100`? Wondering if that would be enough, by adding the lambda to the [execution queue instead of running it immediately](http://www.geekabyte.io/2014/01/javascript-effect-of-setting-settimeout.html)

Reply:
thanks thats even better, passed locally with maxprocs=1, let's see if it will pass on ci

Reply:
yeah, that way it becomes more explicit that it's not dependent on any amount of time, just reordering of events.